includes/class-press-sync.php

```diff
@@ -46,6 +54,7 @@ class Press_Sync {
 	public function __construct() {
 		// Initialize plugin classes.
 		$this->plugin_classes();
+		$this->delta_date = date( 'Y-m-d 00:00:00', strtotime( get_option( 'ps_delta_date' ) ) ) ?: false;
```

Comment:
If it's already in the database we probably have bigger problems, but we're not sanitizing the ps_delta_date value here before we're using it and passing it to the date method. It'll probably make your date() method flunk out of class long before it gets appended as WHERE condition in your future SQL, but it's worth double-checking this to make sure there are no security implications here.

Reply:
Considering `date` returns `false` on any non-numeric data, I don't think it's a huge concern, however, it's enough that I agree that sanitization for sanity's sake isn't a bad idea. I'll add that in.